### PR TITLE
Sections oli

### PR DESCRIPTION
**Pull request by oliver-jenny**
### What?

New Sections implemented

### Why?

More options ingame

### How?

Magic 🪄

### Testing?

WIP

### Anything Else?

closes: #44 
closes: #47 
closes: #81 
closes: #127 
closes: #137 
closes: #141 

[//]: # (for a sample see: https://www.pullrequest.com/blog/writing-a-great-pull-request-description/)

### CODE REVIEW

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section614.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    public SectionNode getBody(PlayerDto current) {
+        return SectionNode.root()
+                .text("""
+                        The  Stinking  River  has  cut  its  way  through  the  high  grounder.
+                        On the edge of the chasm that overlooks the river lies the village  of  High  Therys.
+                        Just outside of town, three bodies hang on a gallows, slowly rotting.
+                        Inside, the villagers are having a fete. They welcome you. You can get some rest
+                        and recuperation here. When you are ready, you leave.
+                        """)
+                .clickable(a -> a.addStamina(5), b -> b.text("Add up to 5 stamina"))
```
Comment (keykey7) [inline fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section614.java:26]: note: atm this is clickable multiple times (so you can recover more than just 5HP). -> try it in the CLI.
@Lion-alt has some hacks for this scenario in his PR. :)

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section618.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    @Override
+    public SectionNode getBody(PlayerDto current) {
+        return SectionNode.root()
+                .text("""
+                        Becoming an initiate of Lacuna gives you the benefit of paying
+                        less for blessings and other services the temple can offer. It costs
+                        30 Shards to become an initiate. You cannot do this if you are
+                        already an initiate of another temple.
+                        """)
+                .activeIf(!current.god().isEmpty(), a -> a.text("Become an initiate of Lacuna")
```
Comment (keykey7) [inline fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section618.java:27]: I think no god is `==null`...?

and current shards >= 30

**Review by keykey7 — APPROVED** [review]

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section614.java (hunk 1 above)
Comment (oliver-jenny) [reply fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section614.java:26]: opened an issue

**Review by Lion-alt — APPROVED** [review]
LGTM 👍 Fighting could be implemented with a simple check if the current player has more defence than required. This is enough for now.

**Review by Desi7 — APPROVED** [review]
Good work 👍

**Review by kimberlymo — APPROVED** [review]
lgtm 👍

**Review by MagischerWolf05 — APPROVED** [review]
LGTM

**Review by Nilstrieb — APPROVED** [review]

**Pull request merged**